packages/e2e-tests/register/models/SetupSeedPhrase.js

```diff
@@ -0,0 +1,16 @@
+class SetupSeedPhrasePage {
+    constructor(page) {
+        this.page = page;
+    }
+    async navigate(accountId) {
+        await this.page.goto(`/setup-seed-phrase/${accountId}/phrase`);
+    }
+    async copySeedPhrase() {
+        await this.page.click(`button:text-matches("Copy Phrase", "i")`);
+        return this.page.evaluate(() => navigator.clipboard.readText());
+    }
+    async continueToSeedPhraseVerification() {
+        await this.page.click(`button:text-matches("Continue", "i")`);
```

Comment:
test id here also please; if we rename the UI display for this button to "Next" but it is logically the same button, we don't want the test suite to fail